### DISCUSSION TITLE
Cannot connect

### DISCUSSION THREAD

**Discussion by whomever57** [audiamus/BookLibConnect#31] [General]
Failed to connect. Says web page does not exist trying to setup initial profile. Any suggestions?

**Comment by whomever57** [comment]
<html>
<body>


  | Looking for Something?We're sorry. The Web address you entered is not a functioning page on our site Go to Amazon.com's Home Page
-- | --



</body>
</html>Looking for Something?
We're sorry. The Web address you entered is not a functioning page on our site
 Go to Amazon.com's [Home](https://www.amazon.com/) Page

**Comment by audiamus** [comment]
That's the envisaged behaviour, You have to copy the URL of that final page into the empty text box of the Book Lib Connect dialog. The tool tip popping up when creating the initial log-in URL also mentions this.

Book Lib Connect will never see your Amazon/Audible user name or password.

**Reply by whomever57** [reply]
Thank you!
